src/astx/packages.py

```diff
@@ -148,3 +149,127 @@ def __init__(
     def __str__(self) -> str:
         """Return the string representation of the object."""
         return f"PROGRAM[{self.name}]"
+
+
+@public
+class AliasExpr(Expr):
+    """Represents an alias in an import statement."""
+
+    name: str
+    asname: Optional[str]
+
+    def __init__(
+        self,
+        name: str,
+        asname: Optional[str] = None,
+        loc: SourceLocation = NO_SOURCE_LOCATION,
+        parent: Optional[ASTNodes] = None,
+    ) -> None:
+        super().__init__(loc=loc, parent=parent)
+        self.name = name
+        self.asname = asname
+        self.kind = ASTKind.AliasExprKind
+
+    def __str__(self) -> str:
+        """Return a string representation of the alias."""
+        if self.asname:
+            return f"{self.name} as {self.asname}"
+        else:
+            return self.name
+
+    def get_struct(self, simplified: bool = False) -> ReprStruct:
+        """Return the AST structure of the alias."""
+        key = "Alias"
+
+        name_dict = {"name": self.name}
+        asname_dict = {"asname": self.asname} if self.asname else {}
+        value: ReprStruct = {
+            **name_dict,
+            **asname_dict,
+        }
+        return self._prepare_struct(key, value, simplified)
+
+
+@public
+class ImportStmt(StatementType):
+    """Represents an import statement."""
+
+    names: list[AliasExpr]
+
+    def __init__(
+        self,
+        names: list[AliasExpr],
+        loc: SourceLocation = NO_SOURCE_LOCATION,
+        parent: Optional[ASTNodes] = None,
+    ) -> None:
+        super().__init__(loc=loc, parent=parent)
+        if isinstance(names, list):
```

Comment:
please remove this `isinstance` validation, because it will be done via typeguard